plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -79,130 +61,7 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		return nil
 	}
 
-	var volumeLabels map[string]string
-	if volume.Spec.AWSElasticBlockStore != nil {
-		labels, err := l.findAWSEBSLabels(volume)
-		if err != nil {
-			return admission.NewForbidden(a, fmt.Errorf("error querying AWS EBS volume %s: %v", volume.Spec.AWSElasticBlockStore.VolumeID, err))
-		}
-		volumeLabels = labels
-	}
-	if volume.Spec.GCEPersistentDisk != nil {
-		labels, err := l.findGCEPDLabels(volume)
-		if err != nil {
-			return admission.NewForbidden(a, fmt.Errorf("error querying GCE PD volume %s: %v", volume.Spec.GCEPersistentDisk.PDName, err))
-		}
-		volumeLabels = labels
-	}
-
-	if len(volumeLabels) != 0 {
-		if volume.Labels == nil {
-			volume.Labels = make(map[string]string)
-		}
-		for k, v := range volumeLabels {
-			// We (silently) replace labels if they are provided.
-			// This should be OK because they are in the kubernetes.io namespace
-			// i.e. we own them
-			volume.Labels[k] = v
-		}
-	}
+	volume.Status.Phase = api.VolumePending
```

Comment:
I'll look into if we can do that, but I don't see why we couldn't. Removing the admission plugin completely would break existing configuration that uses it wouldn't it?